libs/shared/barista-definitions/src/lib/barista-definitions.ts

```diff
@@ -172,29 +174,31 @@ export interface BaStrapiCTA extends BaStrapiBase {
   buttonlink: string;
 }
 
-export interface BaOverviewPageContents extends BaPageMetaBase {
-  sections: BaOverviewPageSection[];
+export interface BaCategoryNavigationContents extends BaPageMetaBase {
+  sections: BaCategoryNavigationSection[];
 }
 
-export interface BaOverviewPageSection {
+export interface BaCategoryNavigationSection {
   title?: string;
-  items: BaOverviewPageSectionItem[];
+  sectiontitle?: string;
```

Comment:
Why do we need a title and a section title?

Reply:
If we need it, think about renaming it to `sectionTitle`.

Reply:
You're right we don't really need it... Thanks!